src/tests/ftest/util/daos_utils.py

```diff
@@ -410,6 +418,27 @@ def container_destroy_snap(self, pool, cont, snap_name=None, epoch=None,
             CommandFailure: if the daos pool query command fails.
 
         """
+        if len(epochs) == 1:
+            return self._get_result(
+                ("container", "destroy-snap"), pool=pool, svc=svc, cont=cont,
+                sys_name=sys_name, snap=snap_name, epc=epochs[0])
+
+        epcrange = "{}-{}".format(epochs[0], epochs[1])
         return self._get_result(
             ("container", "destroy-snap"), pool=pool, svc=svc, cont=cont,
-            sys_name=sys_name, snap=snap_name, epc=epoch, )
+            sys_name=sys_name, snap=snap_name, epcrange=epcrange)
+
+    def container_list_snaps(self, pool, cont, svc=None):
+        """List snapshot in a container.
+
+        Args:
+            pool (str): Pool UUID.
+            cont (str): Container UUID.
+            svc (str): Service replicas. Defaults to None.
+
+        Returns:
+            CmdResult: Object that contains exit status, stdout, and other
+                information.
+        """
+        return self._get_result(
+            ("container", "list-snaps"), pool=pool, cont=cont, svc=svc)
```

Comment:
As we add new methods we shouldn't be making use of the deprecated get_output method and this output parsing should be moved inside of the new method being added.

Reply:
Done